autoscale-scaler-marathon/pom.xml

```diff
@@ -75,6 +66,14 @@
             <artifactId>mockito-core</artifactId>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>org.apache.httpcomponents</groupId>
```

Comment:
Why are these 2 dependencies listed below the test dependencies? Test dependencies should be listed last.

Reply:
Fixed